z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -111,13 +111,63 @@ Provision a Red Hat machine based on the following specifications, or using exis
 **Requirements:**
 
 * Python3
-* Ansible
+* Ansible >=2.8
 * jq
+* wget
+* git
+* firewalld
 * Python modules required in the playbooks. Namely:
   * openstackclient 
   * openstacksdk >= 0.57.0
   * netaddr
 
+**Install:**
+
+Use the following command to register the system, then automatically associate any available subscription matching that system:
+```sh
+sudo subscription-manager register --username <username> --password <password> --auto-attach
+```
+After registration, use the following command to enable ansible repository, or use newer version. 
+Note: Our scenario has been tested on Ansible 2.8.18 version. 
+```sh
+sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms 
+```
+Install the packages from the repository in the system:
+```sh
+sudo yum install python3 ansible jq wget git firewalld -y
+```
+Upgrade the pip package and dnf:
+```sh
+sudo -H pip3 install --upgrade pip
+sudo dnf update -y
+```
+Install the required package through dnf:
+```sh
+sudo dnf install redhat-rpm-config gcc libffi-devel python3-devel openssl-devel cargo -y
+```
+Then create the requirements file and use pip3 to install the python modules:
+```sh
+cat <<'EOF' >> requirements.txt 
+# The order of packages is significant, because pip processes them in the order
+# of appearance. Changing the order has an impact on the overall integration
+# process, which may cause wedges in the gate later.
+pbr!=2.1.0,>=2.0.0 # Apache-2.0
+
+cliff>=3.5.0 # Apache-2.0
+iso8601>=0.1.11 # MIT
+openstacksdk>=0.57.0 # Apache-2.0
+osc-lib>=2.3.0 # Apache-2.0
+oslo.i18n>=3.15.3 # Apache-2.0
+oslo.utils>=3.33.0 # Apache-2.0
+python-keystoneclient>=3.22.0 # Apache-2.0
+python-novaclient>=17.0.0 # Apache-2.0
+python-cinderclient>=3.3.0 # Apache-2.0
+stevedore>=2.0.1 # Apache-2.0
+EOF
+
```

Comment:
The above requirements have versions information. Those versions are for python-openstackclient latest version (5.5).  Which RHEL version we tested this requirements? Let's clarify it too.
For example:
The requirements.txt are tested for python-openstackclient =5.5.0 on RHEL OS version 8.2.

Reply:
Sure, I've update it.